src/style.css

```diff
@@ -1,3 +1,62 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  margin-top: 3px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+}
+
+.nav__link {
+  margin-right: 20px;
+}
+
+.nav__link:last-child {
+  margin-right: 0;
+}
+
+.nav__item {
+  display: flex;
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', serif;
+  color: black;
+  height: 60px;
+  font-weight: 500;
+  font-size: 12px;
+  text-decoration: none;
+  text-align: center;
+  text-transform: uppercase;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav__item:hover {
+  color: #00acdc;
```

Comment:
The hover color for `.nav__item` is set to #00acdc. Verify that this color provides enough contrast with the background to meet accessibility standards.